server/notebooks/api_views.py

```diff
@@ -145,5 +147,6 @@ def perform_create(self, serializer):
                     f"Based on non-latest revision {parent_revision_id} "
                     f"(expected: {last_revision.id})"
                 )
-
+        ctx["is_draft"] = True
         serializer.save(**ctx)
```

Comment:
(manually changed this before merging, no need to do anything here)

Reply:
`ctx` is a local dict (`get_serializer_context()` always generate a new dict `return {"notebook_id": notebook_id}`), so I thought it might be okay to update it.

Reply:
Yeah, this is partly just a stylistic preference-- I like to treat stuff returned from other functions as immutable, where possible (imitates having a const keyword).